core/pkg/filestream/historyupdate.go

```diff
@@ -40,9 +41,25 @@ func (u *HistoryUpdate) Chunk(fs *fileStream) error {
 			"filestream: failed to serialize history: %v", err)
 	}
 
-	fs.addTransmit(&TransmitChunk{
-		HistoryLines: []string{string(line)},
-	})
+	if len(line) > maxFileLineBytes {
+		// We consider this non-blocking. We'll upload a run with some missing
+		// data, but it's better than not uploading anything at all, as long
+		// as we inform the user.
+		//
+		// TODO: inform the user
+		fs.logger.CaptureWarn(
+			"filestream: run history line too long, skipping",
```

Comment:
is there an easy way to have the info of which line caused the issue, when we look at the debug log, we will know that we dropped some data, but not what it was. 
if there isn't this is good enough

Reply:
i guess same for all the other types of lines we drop.

Reply:
probably shouldn't print it to logs, considering it's at least 10MB